src/main/java/org/openrewrite/staticanalysis/EqualsAvoidsNullVisitor.java

```diff
@@ -97,8 +92,8 @@ private static J.Binary literalsFirstInComparisonsNull(J.MethodInvocation m, Exp
                 m.getPrefix(),
                 Markers.EMPTY,
                 requireNonNull(m.getSelect()),
-                build(J.Binary.Type.Equal).withBefore(SINGLE_SPACE),
-                firstArgument.withPrefix(SINGLE_SPACE),
+                JLeftPadded.build(J.Binary.Type.Equal).withBefore(Space.SINGLE_SPACE),
```

Comment:
Build maybe okay, but the Space is DRY Space. SPACE 

If i need unwanted context i move my mouse and not clutter the code with redundancy, but its a style where most dev´s use full qualified imports.

![image](https://github.com/user-attachments/assets/162d7379-7233-4540-958c-ab39c9bd7870)

Reply:
Often classes use a mix of the [two constants in Space](https://github.com/openrewrite/rewrite/blob/0e7364eb9d8d6b6e7f07e9da190b8fe375ff31ab/rewrite-java/src/main/java/org/openrewrite/java/tree/Space.java#L32-L34), only one of which has `_SPACE` in the name for the repetition. I'd prefer then to be consistent with prior use, and i/fwhen we dedice to change that, do so through [our enforced patterns in rewrite-recommendations](https://github.com/openrewrite/rewrite-recommendations/commit/88183fbd237bce56abc44a97def111ad9faf703a). I hope that context helps!

Reply:
I suspect the idea with the name was to emphasize that the object represents a single space character. The class `Space` represents any sequence of whitespace and comments, so `Space.SINGLE` would be a bit ambiguous (a single space, tab, newline, or comment?), so `Space.SINGLE_SPACE` uses "space" in two different meanings.